## Prueba #0201 

In [5]:
import pandas as pd
import os

#### Leer los datos.

In [6]:
# Ir al directorio en el que se encuentra el archivo. 
os.chdir(r'./')                # Reemplazar por la ruta correcta.

In [7]:
df = pd.read_csv('data_studentlist.csv', header='infer')

In [10]:
df.head(5)

name gender  age  grade absence bloodtype  height  weight
0    Jared Diamond      M   23      3       Y         O   165.3    68.2
1   Sarah O'Donnel      F   22      2       N        AB   170.1    53.0
2     Brian Martin      M   24      4       N         B   175.0    80.1
3     David Hassel      M   23      3       N        AB   182.1    85.7
4  Clara Rodriquez      F   20      1       Y         A   168.0    49.5

#### Responda a las preguntas utilizando la librería Pandas.

1). Altura media de los alumnos:

In [12]:
#df = pd.DataFrame(lists, columns = ['name', 'gender', 'age', 'grade','absence', 'bloodtype', 'height', 'weight'])
df.mean()

age        22.352941
grade       2.294118
height    170.035294
weight     61.064706
dtype: float64

2). Altura media de las alumnas:

3). Peso medio de los alumnos:

4). Peso medio de las alumnas:

5). La altura más alta entre los alumnos:

6). La altura más baja entre las alumnas:

7). El peso menor entre los alumnos de estatura superior a 175 cm:

8). El peso mayor entre las alumnas de altura inferior a 160 cm:

9). La calificación promedio de los estudiantes sin ausencia ('N'):

10). La calificación promedio de los estudiantes con ausencia ('Y'):

11). Altura media de los estudiantes con tipo de sangre 'A' o 'AB':

12). Altura promedio de los estudiantes **masculinos** con tipo de sangre 'A' o 'AB':

13). Edad promedio de los estudiantes con ausencia ('Y') cuya calificación es igual o mayor a 3: